common/src/main/scala/explore/components/FormStaticData.scala

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import explore.components.ui.GPPStyles
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.TagMod
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common._
+
+/**
+ *  A control similar to a readonly FormInput styled as `static-data`,
+ *  except that this control will be only as wide as it needs to be.
+ */
+
+final case class FormStaticData(
+  id:        String,
+  value:     String,
+  label:     String,
+  hideLabel: Boolean = false,
+  modifiers: Seq[TagMod] = Seq.empty
+) extends ReactProps[FormStaticData](FormStaticData.component) {
+  def apply(mods: TagMod*): FormStaticData = copy(modifiers = modifiers ++ mods)
+}
+
+object FormStaticData {
+  type Props = FormStaticData
+
+  val component =
+    ScalaComponent
+      .builder[Props]("FormStaticData")
+      .render_P { props =>
+        <.div(
+          props.modifiers.toTagMod,
+          (GPPStyles.HideLabel).when(props.hideLabel),
+          ^.cls := "field",
+          <.label(props.label, ^.htmlFor := props.id),
+          <.div(^.cls := "ui input",
```

Comment:
Would it be possible to use SUI controls? `Label` and `Input`?

Reply:
> Would it be possible to use SUI controls? `Label` and `Input`?

Probably. I'll see what I get.

Reply:
>  I might be able to use an SUI `Label`.
It turns out the using a Label complicates things more than I thought. It adds the `ui label` classes to the label, which styles it unlike the labels for the other Form* controls. Apparently, if you pass a string (not a Label) to a Form* control in the `label` parameter, it doesn't actually use a an SUI label. I could make the `label` parameter here be a `String | Label` union to allow usage of an SUI Label if desired.